README.md

```diff
@@ -79,6 +94,15 @@ resources               clusterpedia.io/v1beta1  false   Resources
 More information about [Search Conditions](https://clusterpedia.io/docs/usage/search/),
 about [Label Selector](https://clusterpedia.io/docs/usage/search/#label-selector) and [Field Selector](https://clusterpedia.io/docs/usage/search/#field-selector)
 
+## Usage Samples
+We can search for resources configured in *PediaCluster*, Clusterpedia supports two types of resource search:
+* Search for resources that are compatible with Kubernetes OpenAPI
+* Search for [`Collection Resource`](https://clusterpedia.io/docs/concepts/collection-resource/)
+```sh
+$ kubectl api-resources | grep clusterpedia.io
+collectionresources     clusterpedia.io/v1beta1  false   CollectionResource
+resources               clusterpedia.io/v1beta1  false   Resources
+```
 ### Search in Kubernetes OpenAPI Compatible Way
 Although we can use URL to search resources, if we want to use `kubectl` to query more conveniently, 
 we need to [configure the cluster shortcut for `kubectl`](https://clusterpedia.io/docs/usage/access-clusterpedia/#configure-the-cluster-shortcut-for-kubectl).
```

Comment:
you [configured shortcuts for `kubectl`](https://clusterpedia.io/docs/usage/access-clusterpedia/#configure-the-cluster-shortcut-for-kubectl).